infra/cifuzz/fuzz_target.py

```diff
@@ -111,68 +113,57 @@ def _download_corpus(self):
   def prune(self):
     """Prunes the corpus and returns the result."""
     self._download_corpus()
-    prune_options = [
-        '-merge=1', self.pruned_corpus_path, self.latest_corpus_path
-    ]
-    result = self.fuzz(use_corpus=False, extra_libfuzzer_options=prune_options)
-    return FuzzResult(result.testcase, result.stacktrace,
-                      self.pruned_corpus_path)
-
-  def fuzz(self, use_corpus=True, extra_libfuzzer_options=None):
+    with clusterfuzz.environment.Environment(config_utils.DEFAULT_ENGINE,
+                                             self.config.sanitizer,
+                                             self.target_path,
+                                             interactive=True):
+      engine_impl = clusterfuzz.fuzz.get_engine(config_utils.DEFAULT_ENGINE)
+      result = engine_impl.minimize_corpus(self.target_path, [],
```

Comment:
I guess this is a question for CF's API but why do we need to pass target_path if we already do so to environment?

Reply:
The environment requirement is an unfortunate artifact of ClusterFuzz depending on many env vars. The API should avoid this, rather than depend even more on env vars.

Reply:
ACK